README.md

```diff
@@ -171,11 +172,10 @@ A FLEX position can be filled with a RB, WR or TE.
 ### Restricted Free Agency
   - Starting in 2018, owners may designate 0 to 2 players as Restricted Free Agents (RFA)
   - Designated RFAs are set at the soft cap deadline
-  - RFAs may not be designated if acquired in the Free Agent Auction 1 year prior
+  - Drafted players are not eligible for RFA the following year
   - RFA designation permits an owner to retain a player at his "market price" plus $1
   - RFA players will be dispositioned in the first two rounds of the Free Agent Auction draft
   - Owners may not exceed the salary cap during the RFA auction
-  - Players acquired from another owner via RFA reset their RFA clock, i.e., they are not eligible for RFA the following year.
```

Comment:
Yeah, the line `Drafted players are not eligible for RFA the following year` was intended to cover all drafted players.  I updated the line to add an exception for retained RFAs.  I also added a line stating that there is no limit to the number of years a player may be designated/retained as a RFA.